float64 back_wheel_speed
float64 turn_angle
char battery_level
char error_flag
